requirements.txt

```diff
@@ -31,6 +32,8 @@ mypy==0.770
 mypy-extensions==0.4.3
 packaging==20.3
 permission==0.4.1
+Pillow==7.1.0
+pkg-resources==0.0.0
```

Comment:
Please make sure to not include this package in commits. This seems to be a bug of PyCharm or sth and usually breaks requirements download.

Reply:
From what I've gathered it seems to be a bug in the communication between Ubuntu <-> pip. Fixed this in commit 94dc8d0